testing/scripts/snmp-test.zeek

```diff
@@ -87,7 +87,7 @@ function print_snmp_bulk_pdu(pdu: SNMP::BulkPDU)
 	{
 	print fmt("    request_id:      %s", pdu$request_id);
 	print fmt("    non_repeaters:   %s", pdu$non_repeaters);
-	print fmt("    max_repititions: %s", pdu$max_repititions);
+	print fmt("    max_repetitions: %s", pdu$max_repetitions);
```

Comment:
The second change referencing the field we either need to revert, or adapt `init-bare.zeek`, too. It's typo in the record definition in init-bare.zeek.

https://github.com/zeek/zeek/blob/ec3eca05492031d59f121046f377d101d090a06b/scripts/base/init-bare.zeek#L4661-L4666

Do we want to make a breaking change that fixes the typo in the field, or be nice and not change it, or introduce the fixed name and a `&deprecated` attribute for the old one? Maybe open an issue and drop the `repetitions` fix from this PR.

The CI pipeline output shows:
```
  error in /zeek/testing/btest/../scripts/snmp-test.zeek, line 90: no such field in record (pdu$max_repetitions)
```

Here, for example:
https://cirrus-ci.com/task/4854512750428160?logs=test#L258

Reply:
I'd leave this change out for now.

Reply:
Dropped